Dockerfile

```
@@ -7,11 +7,10 @@ RUN apk update && apk add py3-psycopg2 musl-dev \
     nginx supervisor git \
     openssl ca-certificates \
     gcc postgresql-dev \
- && pip install --upgrade pip \
- && pip install virtualenv
+ && pip install --upgrade pip
 
 RUN         pip install -r /app/requirements.txt
-ADD         . /app
+COPY        dataservice docs .git migrations bin manage.py config.py setup.py /app
```

Docker needs that end slash if the dest is a directory 🤷‍♀ 

```suggestion
COPY        dataservice docs .git migrations bin manage.py config.py setup.py /app/
```